mta/fixtures/project_fixture.py

```diff
@@ -2,9 +2,16 @@
 import pytest
 
 
+@pytest.fixture()
```

Comment:
As we are going to use this fixture everywhere better to have a `session` scope.

Reply:
Done